js/src/setup-mc/setup-stepper/setup-paid-ads/setup-paid-ads.js

```diff
@@ -207,6 +209,10 @@ export default function SetupPaidAds() {
 						disabled={ disabledComplete }
 						onClick={ handleCompleteClick }
 						eventName="gla_onboarding_complete_with_paid_ads_button_click"
+						eventProps={ {
+							budget: paidAds.amount,
+							audiences: paidAds.countryCodes.join( ',' ),
```

Comment:
With a fresh starting webpage or clearing a client session `sessionStorage.removeItem('gla-onboarding-show-paid-ads-setup')`, the following error occurs:

![image](https://github.com/woocommerce/google-listings-and-ads/assets/17420811/a062044f-96f6-4f0e-ad29-ad857a967803)